### DISCUSSION TITLE
What should I be uploading by the LittleFS tool?

### DISCUSSION THREAD

**Discussion by Revoxs** [Winston-Lu/ESP8266-LED-Shelf#5] [Q&A] [answered]
I did the configuration in VScode (24HR Clock, Sectrets.h etc.), but I didn't get what to upload by which platform

**Comment by Winston-Lu** [comment, marked as answer]
I assume you are using PlatformIO, so I don't have much experience with LittleFS there. You should be uploading everything in the /data/ folder. By default in the Arduino IDE, LittleFS uploads whatever is in the data folder, with a file such as `index.html` being in the root of the file system (dont make the /data/ folder the root).
